automol/const.py

```diff
@@ -24,17 +24,25 @@ class ReactionClass(str, enum.Enum):
     SUBSTITUTION = "substitution"
 
     def __str__(self):
+        """Construct a string from Reaction Class.
+
+        :return: String from self
+        """
         return self.value
 
     def __repr__(self):
+        """_summary_.
+
+        :return: _description_
+        :rtype: _type_
+        """
```

Comment:
The first one gets called when you call `repr(object)`, whereas the second one gets called when you call `str(object)`.

Reply:
Example:
```
>>> print(repr("abc"))
'abc'
>>> print(str("abc"))
abc
```

Reply:
The way we are using `__repr__()` above, simply returning the string literal, isn't really the intended usage, and I don't think it's really necessary. But I am hesitant to take it out because this class is tied in with some ugly higher-level code that is very bug-prone. Eventually, I would like to get rid of these classes entirely.